src/components/organisms/InformationLeftColumn/InformationLeftColumn.scss

```diff
@@ -2,13 +2,21 @@
 
 .information-left-column-container {
   flex: 0 0 auto;
+  position: relative;
 
   .band-logo {
     max-width: 100%;
     max-height: 100%;
 
     &.expanded {
-      width: 200px;
+      width: 100px;
```

Comment:
It's smaller when expanded?

Reply:
smaller than before but still bigger than when not expanded